apps/widalarmeta/widget.js

```diff
@@ -49,17 +52,34 @@
       g.setFontAlign(0,0); // center fonts
       g.clearRect(this.x, this.y, this.x+this.width-1, this.y+23);
 
-      var text = hours.padStart(2, '0') + ":" + minutes.padStart(2, '0');
+      var digitWidth = 5;
+      var extraPixels = 2;
+      var nrDigits = 5;
+
+      var text = "";
+      if (config.padHours) {
+        text += hours.padStart(2, '0');
+      } else {
+        text += hours;
+        if (hours < 10) {
+          nrDigits -= 1;
```

Comment:
Wouldn't it be easier to calculate the width
from the string length?

Reply:
You are most likely right. I clearly haven't thought enough about it. It was a fixed value before and I just adapted it. I'll change that.

Reply:
@nxdefiant: I used the `g.stringWidth(...)` function instead. With that it is even easier to add further fonts. As an example I added another 4x5 font option.

Reply:
cool and a lot easier to read